BetterJoyForCemu/Config.cs

```diff
@@ -1,9 +1,6 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.IO;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace BetterJoyForCemu {
 	public static class Config { // stores dynamic configuration, including
```

Comment:
Config seems to be slightly different indentation-wise from the other files, strangely.